Python/Product/Analysis/AnalysisValueSetExtensions.cs

```diff
@@ -270,7 +271,7 @@ class DotsLastStringComparer : IComparer<string> {
                     return -1;
                 }
 
-                return x.CompareTo(y);
+                return string.Compare(x, y, StringComparison.CurrentCultureIgnoreCase);
```

Comment:
We can extract `CompareCurrentIgnoreCase` extension method

Reply:
I don't think we use that one often enough to be worth extracting.